doc/source/user/numpy-for-matlab-users.rst

```diff
@@ -231,23 +136,68 @@ General Purpose Equivalents
      - short-circuiting logical OR operator (Python native operator);
        scalar arguments only
 
+   * - ::
+        
+        >> 4 == 4
+        ans = 1
+        >> 4 == 5
+        ans = 0
+
+     - ::
+
+        >>> 4 == 4
+        True
+        >>> 4 == 5
+        False
+
+     - The `boolean objects
+       <https://docs.python.org/3/library/stdtypes.html?highlight=boolean#boolean-values>`_
```

Comment:
Should be able to use an intersphinx link here, perhaps `<python:boolean-values>`

Reply:
> Should be able to use an intersphinx link here, perhaps `<python:boolean-values>`

Nice! That worked, I'll push the change on next commit.